long_range_colloquium.md

```diff
@@ -25,6 +25,12 @@ Attendance is open to everyone, but please [register](#nurit-avraham-weizmann-in
 
 22 July 2020, 1:30 PM ET (19:30 CEST)
 
+> #### Correlating surface spectroscopy with qubit measurements to systematically eliminate sources of noise
+>
+> The nitrogen vacancy (NV) center in diamond exhibits spin-dependent fluorescence and long spin coherence times under ambient conditions, enabling applications in quantum information processing and sensing. NV centers near the surface can have strong interactions with external materials and spins, enabling new forms of nanoscale spectroscopy. However, NV spin coherence degrades within 100 nanometers of the surface, suggesting that diamond surfaces are plagued with ubiquitous defects. Prior work on characterizing near-surface noise has primarily relied on using NV centers themselves as probes; while this has the advantage of exquisite sensitivity, it provides only indirect information about the origin of the noise. I will describe our recent efforts to use X-ray and photoelectron spectroscopies, diffraction techniques, and morphology characterization to understand sources of noise at the diamond surface. By correlating this spectroscopic data with single spin measurements, we have been able to devise new surface processing and termination techniques to stabilize shallow NV centers within 5 nm of the surface with coherence times exceeding 100 μs [(PRX 9, 031052)](https://doi.org/10.1103/PhysRevX.9.031052). Specifically, we are able to demonstrate reversible and reproducible control over the top layer of atoms. These highly coherent, shallow NV centers will provide a platform for sensing and imaging down to the scale of single atoms.
```

Comment:
```suggestion
> The nitrogen vacancy (NV) center in diamond exhibits spin-dependent fluorescence and long spin coherence times under ambient conditions, enabling applications in quantum information processing and sensing. NV centers near the surface can have strong interactions with external materials and spins, enabling new forms of nanoscale spectroscopy. However, NV spin coherence degrades within 100 nanometers of the surface, suggesting that diamond surfaces are plagued with ubiquitous defects. Prior work on characterizing near-surface noise has primarily relied on using NV centers themselves as probes; while this has the advantage of exquisite sensitivity, it provides only indirect information about the origin of the noise. I will describe our recent efforts to use X-ray and photoelectron spectroscopies, diffraction techniques, and morphology characterization to understand sources of noise at the diamond surface. By correlating this spectroscopic data with single spin measurements, we have been able to devise new surface processing and termination techniques to stabilize shallow NV centers within 5 nm of the surface with coherence times exceeding 100 μs [1]. Specifically, we are able to demonstrate reversible and reproducible control over the top layer of atoms. These highly coherent, shallow NV centers will provide a platform for sensing and imaging down to the scale of single atoms.
```

Reply:
resolving: the two versions are identical.